src/screens/Inventory/InventoryMain.tsx

```diff
@@ -0,0 +1,56 @@
+import { createStyles, Fab, Theme, withStyles } from '@material-ui/core';
+import AddIcon from '@material-ui/icons/Add';
+import React from 'react';
+import { connect } from 'react-redux';
+import { Link, RouteComponentProps } from 'react-router-dom';
+import BaseScreen from '../../components/BaseComponents/BaseScreen';
+import BaseScrollView from '../../components/BaseComponents/BaseScrollView';
+import InventoryCard from '../../components/Inventory/InventoryCard';
+import { InventoryRecord, SiteRecord } from '../../lib/airtable/interface';
+import { SiteInventoryData } from '../../lib/redux/inventoryDataSlice';
+import { EMPTY_SITE } from '../../lib/redux/siteDataSlice';
+import { RootState } from '../../lib/redux/store';
+
+const styles = (theme: Theme) =>
+  createStyles({
+    fab: {
+      position: 'absolute',
+      bottom: theme.spacing(1),
+      right: theme.spacing(2),
+      color: 'white',
+    },
+});
+
+interface InventoryProps extends RouteComponentProps {
+  classes: { fab: string };
+  sitesInventory: Record<string, SiteInventoryData>;
+  currentSite: SiteRecord;
+}
+
+function InventoryMain (props: InventoryProps) {
+    const { classes, sitesInventory, currentSite } = props;
+    const currentSiteInventory = sitesInventory[currentSite.id];
+    return (
+      <BaseScreen title="Inventory">
+        <BaseScrollView>
+          {currentSiteInventory.siteInventory.map((inventory: InventoryRecord) =>  (
+            <Link key={inventory.id} to={{ pathname: `/inventory/item`, state: { inventoryItem: inventory }}}>
```

Comment:
Right now, key is dubious since ids are not unique (all offline will be empty string). Let's do the negative `id` for offline items? The more I think about different cases the more I like the idea